### PR TITLE
fix: fix the `CURRENT_TAG` in workflows/update-and-rebuild.yaml

### PR DESCRIPTION
**Pull request by kokeshiM0chi**
# Overview

`CURRENT_TAG` must indicate the current finschia-wasm version, i.e. version of Finschia/finschia.
 I wrote "https://api.github.com/repos/Finschia/finschia-proto/tags" as a sample code and forgot to change it to the correct code.
In this pr, fix to manage a finschia tag in the finschia-wasm env file and use it for automatic updates.

### CODE REVIEW

File: .github/workflows/update-and-rebuild.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     steps:
       - name: Set current tag to env
         run: |
-          CURRENT_TAG=$(curl  "https://api.github.com/repos/Finschia/finschia-proto/tags" | jq -r '.[0].name')
+          CURRENT_TAG=$(curl https://raw.githubusercontent.com/Finschia/finschia-wasm/main/env | grep FINSCHIA_VERSION | awk -F "=" '{print $2}' )
```
Comment (170210) [inline .github/workflows/update-and-rebuild.yaml:22]: Why not read the `FINSCHIA_VERSION` in `../env`

File: env [hunk 2]
```diff
@@ -1,3 +1,4 @@
+FINSCHIA_VERSION="v2.0.1"
 FINSCHIA_SDK_VERSION="v0.48.1"
 WASMD_VERSION="v0.2.0"
 OSTRACON_VERSION="v1.1.2"
```
Comment (zemyblue) [inline env:4]: When are this versions updated?

Comment (kokeshiM0chi) [reply env:4]: Thank you for your comment. The new version is PR here, but it references cometBFT.
So, close it once, delete the automatically generated branch, and wait for it to be automatically generated the next day.
https://github.com/Finschia/finschia-wasm/pull/22

File: .github/workflows/update-and-rebuild.yaml (hunk 1 above)
Comment (kokeshiM0chi) [reply .github/workflows/update-and-rebuild.yaml:22]: Do not checkout finschia-wasm when checking for updates.
env file cannot be viewed unless checked out.

Comment (loloicci) [reply .github/workflows/update-and-rebuild.yaml:22]: Why does this not checkout itself?

Comment (kokeshiM0chi) [reply .github/workflows/update-and-rebuild.yaml:22]: There is no need to run finschia-wasm unless the finschia tag is newer than the finschia tag managed by finschia-wasm.

Comment (loloicci) [reply .github/workflows/update-and-rebuild.yaml:22]: Sorry, my question is: After this step, Finschia/finschia is gotten with checkout. Why is this step's way not unified with it?

**Review by loloicci — APPROVED** [review]

**Review by 170210 — APPROVED** [review]

**Pull request merged**